auth-server/test/resolvers/mutation/login.spec.ts

```diff
@@ -0,0 +1,66 @@
+import 'mocha'
+import { expect } from 'chai'
+import signup from '../../../src/resolvers/mutation/signup'
+import login from '../../../src/resolvers/mutation/login'
+import User from '../../../src/model/User'
+import { Context } from '../../../src/types'
+import { AuthenticationError } from 'apollo-server'
+
+describe('signup mutation', () => {
+	let signupResult
+	let fakectx: Context = {
+		req: {},
+		res: {
+			cookie: () => {}
+		}
+	} as any
+	const email = 'test@email.com'
+	const password = 'testpass'
+	const username = 'testuser'
+	const name = 'test name'
+	// const salt = 'testsalt'
+
+	before(async () => {
+		signupResult = await signup(null, { email, password, username, name }, fakectx)
+	})
+
+	after(async () => {
+		await User
+			.query()
+			.where({ id: signupResult.user.id })
+			.del()
+	})
+
+	it('should be able to login', async () => {
+		const result = await login(null, { username, password }, fakectx)
+
+		expect(result.user.id).to.exist
+		expect(result.user.id).to.a('number')
+		expect(result.user.email).to.equal(email)
+		expect(result.user.name).to.equal(name)
+		expect(result.user.username).to.equal(username)
+		expect(result.token).to.exist
+		expect(result.token).to.be.a('string')
+	})
+
+	it('should throw an error if username doesn\'t exist', async () => {
+		try {
+			await login(null, { username: 'notexist', password }, fakectx)
+		} catch (error) {
+			expect(error).to.exist
+			expect(error).to.be.an.instanceof(AuthenticationError)
+			expect(error.message).to.equal('User not found')
```

Comment:
I wouldn't test the text specifically, this can evolve and would cause the test not to pass for the wrong reason. I think AuthenticationError is good enough.